### PR TITLE
Aggregate events to edit

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/support/issues/271
Reverts: #740

Upstream fix pending.

Adds write permissions only, since read is aggregated from view already.

### CODE REVIEW

**Comment by tumido** [comment]
cc @4n4nd @larsks @HumairAK

**Comment by HumairAK** [comment]
> Adds write permissions only, since read is aggregated from view already.
Wouldn't you want edit to also have read perms so it can work as a standalone role, and not require view?

Also -- why not aggregate to admin (since that's the cluster role we bind to during onboarding)?

File: cluster-scope/overlays/moc/common/kustomization.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ resources:
   - ../../../base/user.openshift.io/groups/tufts-dcc-6
   - ../../../base/user.openshift.io/groups/uky-redhat
   - ../../../base/user.openshift.io/groups/workshops
+  - ../../../base/rbac.authorization.k8s.io/clusterroles/events-aggregate-to-edit
```
Comment (HumairAK) [inline cluster-scope/overlays/moc/common/kustomization.yaml:41]: mind placing it alphabetically?

**Comment by tumido** [comment]
> Wouldn't you want edit to also have read perms so it can work as a standalone role, and not require view?
>
> Also -- why not aggregate to admin (since that's the cluster role we bind to during onboarding)?

@HumairAK 

That's not how it works. All permissions aggregated to view are aggregated to edit and admin as well. Analogically all permission aggregated to edit are also aggregated/propagated to admin. So if view role has read on events, so does edit and admin. And this PR adds write for events to edit which means it's added to admin as well. See my upstream PR that did the same and got merged into Kubernetes: https://github.com/kubernetes/kubernetes/pull/34729

File: cluster-scope/overlays/moc/common/kustomization.yaml (hunk 1 above)
Comment (tumido) [reply cluster-scope/overlays/moc/common/kustomization.yaml]: done

**Comment by HumairAK** [comment]
>  this PR adds write for events to edit which means it's added to admin as well.  

Gotcha, then

/lgtm
/approve

**Pull request merged**